cmd/theatre-envconsul/acceptance/acceptance.go

```diff
@@ -190,9 +173,75 @@ func (r *Runner) Prepare(logger kitlog.Logger, config *rest.Config) error {
 	return nil
 }
 
+// This should be in testdata, but right now our test runner doesn't support relative file
+// access. We should aim to bring back the ability to run acceptance tests from the ginkgo
+// wrapper.
+const rawPodYAML = `
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: read-a-secret-
+  namespace: staging # provisioned by the acceptance kustomize overlay
+spec:
+  serviceAccountName: secret-reader
+  restartPolicy: Never
+  volumes:
+    - name: theatre-envconsul-serviceaccount
+      projected:
+        sources:
+        - serviceAccountToken:
+            path: token
+            expirationSeconds: 900
+  containers:
+    - name: print-env
+      image: theatre:latest
+      imagePullPolicy: Never
+      env:
+        - name: VAULT_RESOLVED_KEY
+          value: vault:secret/data/kubernetes/staging/secret-reader/jimmy
+      command:
+        - /usr/local/bin/theatre-envconsul
+      args:
+        - exec
+        - --vault-address=http://vault.vault.svc.cluster.local:8200
+        - --no-vault-use-tls
+        - --install-path=/usr/local/bin
+        - --service-account-token-file=/var/run/secrets/kubernetes.io/vault/token
+        - --
+        - env
+      volumeMounts:
+        - name: theatre-envconsul-serviceaccount
+          mountPath: /var/run/secrets/kubernetes.io/vault
+`
+
+const annotatedPodYAML = `
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: read-a-secret-
+  namespace: staging # provisioned by the acceptance kustomize overlay
+  annotations:
+    envconsul-injector.vault.crd.gocardless.com/configs: app
+spec:
+  serviceAccountName: secret-reader
+  restartPolicy: Never
+  containers:
+    - name: app
```

Comment:
think this should be `print-env` to match the YAML above, but this shouldn't affect the test

Reply:
Yep good point, I'll change them to match (though I'll go with app in both cases, as that's what we like to use)